NOTIFY/events.py

```diff
@@ -117,17 +135,16 @@ def reminder(self):
             for names in birthday_list:
                 birthday_msg = random.choice(birthday_greeting).format(names)
                 birthday_img = None
-                chat = pc(self.mode)
                 status_code = chat.send_message(birthday_msg,birthday_img)
```

Comment:
btw disini gua ga liat lu create instance ```chat```, dimana ya? soalnya ```self.chatter``` harusnya, atau?

Reply:
Oiya ini kyny blm diganti, thank you thank you

Reply:
No Prob, paling ga bisa liat kalo ada bug